test/parser.ts

```diff
@@ -1138,6 +1138,28 @@ hoge`;
 		});
 	});
 
+	describe('nomfm', () => {
+		it('multiple line', () => {
+			const input = 'a\n<plain>\nHello\nworld\n</plain>\nb';
+			const output = [
+				TEXT('a\n'),
+				NOMFM('Hello\nworld'),
+				TEXT('\nb')
+			];
+			assert.deepStrictEqual(mfm.parse(input), output);
+		});
+
+		it('single line', () => {
+			const input = 'a\n<plain>Hello world</plain>\nb';
+			const output = [
+				TEXT('a\n'),
+				NOMFM('Hello world'),
+				TEXT('\nb')
+			];
+			assert.deepStrictEqual(mfm.parse(input), output);
+		});
+	});
+
```

Comment:
構文が無効になることをテストした方がよさそう
```suggestion
	describe('nomfm', () => {
		it('multiple line', () => {
			const input = 'a\n<plain>\n**Hello**\nworld\n</plain>\nb';
			const output = [
				TEXT('a\n'),
				NOMFM('**Hello**\nworld'),
				TEXT('\nb')
			];
			assert.deepStrictEqual(mfm.parse(input), output);
		});

		it('single line', () => {
			const input = 'a\n<plain>**Hello** world</plain>\nb';
			const output = [
				TEXT('a\n'),
				NOMFM('**Hello** world'),
				TEXT('\nb')
			];
			assert.deepStrictEqual(mfm.parse(input), output);
		});
	});

```